plugins/destination/postgresql/client/types.go

```diff
@@ -97,89 +99,74 @@ func (*Client) SchemaTypeToCockroach(t schema.ValueType) string {
 	}
 }
 
-func (c *Client) PgToSchemaType(t string) schema.ValueType {
+func (c *Client) PgToSchemaType(tableName string, columnName string, t string) (schema.ValueType, error) {
 	switch c.pgType {
 	case pgTypeCockroachDB:
-		return c.CockroachToSchemaType(t)
+		return c.CockroachToSchemaType(tableName, columnName, t)
 	default:
-		return c.Pg10ToSchemaType(t)
+		return c.Pg10ToSchemaType(tableName, columnName, t)
 	}
 }
 
-func (*Client) Pg10ToSchemaType(t string) schema.ValueType {
-	switch t {
-	case "boolean":
-		return schema.TypeBool
-	case "bigint", "integer":
-		return schema.TypeInt
-	case "double precision":
```

Comment:
Our query that reads the current schema returns `TypeFloat` as PostgreSQL `real`, so this code always failed on a table that has floats columns. I added both `float` and `real` to the supported types